lib/packagekit-glib2/pk-progress.c

```diff
@@ -1044,6 +1094,18 @@ pk_progress_class_init (PkProgressClass *klass)
 				   G_PARAM_READWRITE);
 	g_object_class_install_property (object_class, PROP_UID, pspec);
 
+	/**
+	 * PkProgress:sender:
+	 *
+         * The D-Bus name of the client that started this transaction.
+         *
+	 * Since: 1.2.6
+	 */
+	pspec = g_param_spec_string ("sender", NULL, NULL,
+				     NULL,
+				     G_PARAM_READWRITE);
+	g_object_class_install_property (object_class, PROP_SENDER, pspec);
```

Comment:
Looks like these properties could be improved to be installed all at once and have more flags set, just as in https://github.com/ximion/appstream/pull/448. But that’s something for a separate PR. This change looks good in the context of the existing code.

Reply:
Yeah, I was actually thinking about that while writing it... But modernizing PK is a pretty huge task (and having this consistent for now felt nicer).
The AppStream bits were most likely actually inspired by PKs code, as I was working on both a lot when AppStream was created ;-)